mobile/android/fastlane/.gitignore

```diff
@@ -0,0 +1 @@
+metadata/
```

Comment:
Thanks @Mnoble-19 ... This is a amazing.
For the metadata folder

1. What are its contents?
2. Where can a developer find the content?
3. Has the readme file been updated to incorporate a description and usage of this folder?

Reply:
Hi @NoahNsimbe , the metadata folder contains images, screenshots of the app that are in the PlayStore. It's created as you initialise the Fastlane framework, but it's not something that's needed for the build action to occur.

Reply:
okay thanks @Mnoble-19